test/v2/mocks/connectors/MockRetrieveForeignPropertyPeriodSummaryConnector.scala

```diff
@@ -19,21 +19,21 @@ package v2.mocks.connectors
 import org.scalamock.handlers.CallHandler
 import org.scalamock.scalatest.MockFactory
 import uk.gov.hmrc.http.HeaderCarrier
+import v2.connectors.RetrieveForeignPropertyPeriodSummaryConnector.Result
 import v2.connectors.{IfsOutcome, RetrieveForeignPropertyPeriodSummaryConnector}
 import v2.models.request.retrieveForeignPropertyPeriodSummary.RetrieveForeignPropertyPeriodSummaryRequest
-import v2.models.response.retrieveForeignPropertyPeriodSummary.RetrieveForeignPropertyPeriodSummaryResponse
 
 import scala.concurrent.{ExecutionContext, Future}
 
 trait MockRetrieveForeignPropertyPeriodSummaryConnector extends MockFactory {
 
-  val mockRetrieveForeignPropertyConnector: RetrieveForeignPropertyPeriodSummaryConnector = mock[RetrieveForeignPropertyPeriodSummaryConnector]
+  val mockConnector: RetrieveForeignPropertyPeriodSummaryConnector = mock[RetrieveForeignPropertyPeriodSummaryConnector]
```

Comment:
can this be renamed back to mockRetrieveForeignPropertyConnector for consistancy